src/style.css

```diff
@@ -1 +1,62 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
move to `html` selector

Reply:
Changed selector `*` to  `body`

Reply:
Also added styles from layout `box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);`

Reply:
I also added styles from the layout `box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);.` I had to comment them out because they wrote **"Unexpected unit length-zero-no-unit"**